conf/objs/profile.c

```diff
@@ -78,6 +78,7 @@ static void profile_delete(struct m0_conf_obj *obj)
 	struct m0_conf_profile *x = M0_CONF_CAST(obj, m0_conf_profile);
 
 	m0_conf_profile_bob_fini(x);
+	m0_free(x->cp_pools.af_elems);
```

Comment:
Can we introduce another API, like:
void m0_fid_arr_free(struct m0_fid_arr *fid_arr);

to free a fid array?
So we can avoid directly calling m0_free() here?